File: SunkiojiDalis/Models/Items/Consumables/Foods/WeaponRarities/CommonFood.cs
namespace SignalRWebPack{
  public class CommonFood: AbstractFood {
    public CommonFood(int id, string sprite, string name, int weight, int quantity, int x, int y, int belongsTo, int health) {
      this.Id = id;
      this.Sprite = sprite;
      this.Name = name;
      this.Weight = weight;
      this.Quantity = quantity;
      this.X = x;
      this.Y = y;
      this.BelongsTo = -1;
      this.Health = health;
    }
  }
}

File: SunkiojiDalis/src/Items/Consumables/Foods/FoodRarities/CommonFood.ts
import { AbstractFood } from '../AbstractFood';

export class CommonFood extends AbstractFood {
  Id: number;
  Sprite: string;
  Name: string;
  Weight: number;
  Quantity: number;
  X: number;
  Y: number;
  BelongsTo = -1;
  Health: number;

  constructor(guid: string, itemData: any) {
    super(guid, itemData);
  }
}